src/components/animated/_click.less

```diff
@@ -2,7 +2,8 @@
   box-shadow: 0px 5px 0px @lightGray;
 
   &:active {
-    box-shadow: 0px 2px 0px @lightGray;
+    transform: translateY(3px);
+    box-shadow: 0px 5px 0px @lightGray;
```

Comment:
Add another rule for focus. We don't want the default box-shadow to appear when "Click buttons" are clicked.

Reply:
ok then shall i just remove box-Shadow property or anything else is needed to be done?

Reply:
change it to the following:
```
  &:active, &:focus {
    box-shadow: 0px 2px 0px @lightGray;
  }

&:active {
    transform: translateY(3px);
}
```

Reply:
ok